etc/easy-coding-standard.yml

```diff
@@ -0,0 +1,7 @@
+imports:
```

Comment:
Replace this file with:
etc/ecs.php
`<?php

declare(strict_types=1);

use Symplify\EasyCodingStandard\Config\ECSConfig;

return static function (ECSConfig $ecsConfig): void {
    $ecsConfig->import(__DIR__ . '/../vendor/bin/ecs.php');
};
`

Reply:
Done.